crates/ra_ide_api/src/lib.rs

```diff
@@ -397,7 +397,7 @@ impl Analysis {
         self.with_db(|db| db.find_all_refs(position))
     }
 
-    /// Returns a short text descrbing element at position.
```

Comment:
_this is a bit off-topic, but perhaps it helps anyone reading this_

I too sometimes struggle with grammar :) A small tip from one non-native English speaker who likes to be correct to any other: there's two plugins that I use to help me avoid spelling issues in my code:

* https://github.com/bartosz-antosik/vscode-spellright
* https://github.com/Jason-Rev/vscode-spell-checker

They complement each other, as one is used in "textual" contexts, such as README's, or code comments, while the other is used to avoid common spelling mistakes in my actual Rust code, and works with CamelCase situations.

Here's my config for both of them, in case it helps anyone:

```json
{
    "spellright.language": ["en-GB"],
    "spellright.groupDictionaries": false,
    "spellright.documentTypes": [
        "markdown",
        "plaintext",
        "rust",
    ],
    "spellright.parserByClass": {
        "rust": {
            "parser": "code"
        }
    },
    "spellright.notificationClass": "information",
    "spellright.configurationScope": "user",
    "spellright.statusBarIndicator": false,

    "cSpell.checkLimit": 2048,
    "cSpell.language": "en-GB",
    "cSpell.showStatus": false,
    "cSpell.enabledLanguageIds": [
        "markdown",
        "plaintext",
        "rust"
    ],
    "cSpell.userWords": [
        "Deserialize",
        "Tuple",
        "alloc",
        "clippy",
        "rustc",
        "rustfmt",
        "struct",
    ],
    "cSpell.languageSettings": [
        {
            "languageId": "rust",
            "ignoreRegExpList": [
                "/\/\/\/? .*/"
            ]
        }
    ]
}
```

It's easy to add words to the `cSpell.userWords` array as you work, using the contextual popup in the editor. I trimmed the list down to a few useful ones for Rust development. The same goes for the spellright spelling updates, but those are stored in `$VSCODE/User/spellright.dict`:

```
clippy
fmt
impl
iter
mut
println
rustc
rustfmt
usize
utf8
vec
```

cc @matklad

Reply:
Yeah, I do have a spelling problem in any language :-) 

I've tried using a cspell check extension, but unfortunately it gives a fair amount of false positives, and in general is pretty distractive:  `F8` brings you not to the next compiler error, but to the one of the dozen spelling mistakes. 

I'd love to have a spellchecker which only checks comments and have close to zeto false positiv rate, but I haven't invstigated this yet.

Reply:
You could start by using spellright, and configure it to only show you spelling mistakes in your code comments.

I agree that it's annoying that cspell gives plenty of false positives, but when I'm working in a project, I fairly quickly add all of the project-specific variable names to the list of ignored spellings, after which things quiet down.

Anyway, I agree, it's not perfect, but it helped me, and is "good enough" for now (especially if you skip cspell and only focus on comments/documentation in Rust).